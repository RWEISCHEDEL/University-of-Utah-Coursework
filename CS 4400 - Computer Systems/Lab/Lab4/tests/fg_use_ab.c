extern int a[];
int b[] = {1, 2, 3};

int f(int i) {
  return a[b[i]];
}

int g(int i) {
  return b[a[i]];
}
