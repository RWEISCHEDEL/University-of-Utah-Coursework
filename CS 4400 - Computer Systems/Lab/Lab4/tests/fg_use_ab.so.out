f
  b
  a
g
  a
  b
